code/src/components/Option.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const Option = ({value}) => {
+    return (
+        <option value={value}>{value}</option>
+    )
+}

```

Comment:
The way you broke down option, input, radiobutton, and select... It's so friggin smart!!!